compendium/ServiceComponent/include/cppmicroservices/servicecomponent/ComponentContext.hpp

```diff
@@ -172,6 +172,26 @@ namespace cppmicroservices
                     return std::static_pointer_cast<T>(sObj);
                 }
 
+                /**
+                 * Returns the service object from the cache matching the passed in
```

Comment:
the cache is an implementation detail that I don't think should be mentioned in the public facing documentation. The user shouldn't care from where the service object is retrieved.